src/utils/RouteUtils.js

```diff
@@ -86,7 +85,7 @@ async function getRoutes(
         start,
         end,
         departureTimeQuery,
-        arriveByBool,
+        arriveBy,
```

Comment:
if arriveByBool is getting changed to arriveBy anyway could it be changed to isArriveBy

Reply:
yea good point